modules/distr-tracing-tempo-install-cli.adoc

```diff
@@ -107,16 +98,7 @@ metadata:
 EOF
 ----
 
-. In the project that you created for the TempoStack instance, create a secret for your object storage bucket by running one of the following commands:
-
-** To create a secret from a YAML file:
-+
-[source,terminal]
-----
-$ oc apply -f <secret_file>.yaml
-----
-
-** To create a secret from standard input:
+. In the project that you created for the TempoStack instance, create a secret for your object storage bucket by running the following command:
```

Comment:
@ishitasequeira, same comment here as above in https://github.com/openshift/openshift-docs/pull/70003/files#r1453527819.